anta/tests/system.py

```diff
@@ -566,3 +566,98 @@ def test(self) -> None:
                 self._check_supervisor(output, supervisor_name="Standby")
             except (KeyError, JSONDecodeError, TypeError) as exc:
                 self.result.is_failure(f"Standby Supervisor - Failed to parse command output - {exc!s}")
+
+
+class VerifyFilePresence(AntaTest):
+    """Verifies the file presence on the flash drive.
+
+    Expected Results
+    ----------------
+    * Success: The test will pass if the device is not under or entering maintenance.
+    * Failure: The test will fail if the device is under or entering maintenance.
+
+    Examples
+    --------
+    ```yaml
+    anta.tests.system:
+      - VerifyFilePresence:
+          filename: script.py
+    ```
+    """
+
+    categories: ClassVar[list[str]] = ["system"]
+    commands: ClassVar[list[AntaCommand | AntaTemplate]] = [AntaTemplate(template="dir {flash_memory}", revision=1, ofmt="text")]
```

Comment:
Let's use JSON and update the logic.
```suggestion
    commands: ClassVar[list[AntaCommand | AntaTemplate]] = [AntaTemplate(template="dir {flash_memory}", revision=1, ofmt="json")]
```

Reply:
As discussed, please use the JSON output and skip `cEOSCloudLab`.